utils/slither_format/format_naming_convention.py

```diff
@@ -0,0 +1,600 @@
+                                (new_str_of_interest, num_repl) = re.subn("new"+r'(.*)'+name, "new"+r'\1'+name[0].upper()+name[1:], old_str_of_interest, 1)
+                                if num_repl != 0:
+                                    patch = {
+                                        "detector" : "naming-convention (contract new object)",
+                                        "start" : node.source_mapping['start'] + m.span()[0],
+                                        "end" : node.source_mapping['start'] + m.span()[1],
+                                        "old_string" : old_str_of_interest,
+                                        "new_string" : new_str_of_interest
+                                    }
+                                    if not patch in patches[in_file]:
+                                        patches[in_file].append(patch)
+                                else:
+                                    logger.error(red("Could not find new object?!"))
+                                    sys.exit(-1)
+
+            else:
+                # Ignore contract definition
+                continue
+            
+    @staticmethod
+    def create_patch_modifier_definition(slither, patches, name, contract_name, in_file, modify_loc_start, modify_loc_end):
+        for contract in slither.contracts:
+            if contract.name == contract_name:
+                for modifier in contract.modifiers:
+                    if modifier.name == name:
+                        in_file_str = slither.source_code[in_file]
+                        old_str_of_interest = in_file_str[modify_loc_start:modify_loc_end]
+                        m = re.match(r'(.*)'+"modifier"+r'(.*)'+name, old_str_of_interest)
+                        old_str_of_interest = in_file_str[modify_loc_start:modify_loc_start+m.span()[1]]
+                        (new_str_of_interest, num_repl) = re.subn(r'(.*)'+"modifier"+r'(.*)'+name, r'\1'+"modifier"+r'\2'+name[0].lower()+name[1:], old_str_of_interest, 1)
+                        if num_repl != 0:
+                            patch = {
+                                "detector" : "naming-convention (modifier definition)",
+                                "start" : modify_loc_start,
+                                "end" : modify_loc_start+m.span()[1],
+                                "old_string" : old_str_of_interest,
+                                "new_string" : new_str_of_interest
+                            }
+                            if not patch in patches[in_file]:
+                                patches[in_file].append(patch)
+                        else:
+                            logger.error(red("Could not find modifier?!"))
+                            sys.exit(-1)
+                            
+    @staticmethod
+    def create_patch_modifier_uses(slither, patches, name, contract_name, in_file):
+        for contract in slither.contracts:
```

Comment:
Consider using `slither.get_contract_from_name(contract_name)`

`get_contract_from_name` can be probably also be used in other parts of the PR

It's going to be easier to maintain. The only caveat is that it can return a `None`, but I suppose that it would be a a bug in this context?

Reply:
Ah, didn't know about this function. Done.